# License plate number - 9 alphanumeric char
license_plate: DOT-10003

# State short name - 2 alphabet char
state_short_name: VA

# Carrier Name - 25 alphanumeric char
carrier_name: FMCSA Tech Division

# Carrier ID - (USDOT + space + 7digits)
carrier_id: DOT 1

# Weight: ###,### lbs. (6 number format)
weight: 65000

# Date of Last State Inspection: YYYY-MM-DD
date_of_last_state_inspection: 2020.01.01

# Date of Last ADS Calibration: YYYY-MM-DD
date_of_last_ads_calibration: 2020.02.01

# Pre-Trip/ADS Health Check: : (Green/Red icon)
# This is a self check and startup sequence performed by the driver in the morning
pre_trip_ads_health_check: Green

#ISS Score: ### (number format + font color)
#Color coding on the # only. 
#1-49 (Green font) (Pass - Inspection Not Warranted)
#50-74 (Yellow font) (Optional- Inspection Warranted – Next Level Priority)
#75-100 (Red font) - (Inspect – Inspection warranted – top priority)
iss_score: 25

#Permit Required: (Yes/No)  
permit_required: false
